lib/fetch_googletest.sh

```diff
@@ -1,18 +1,22 @@
 #! /bin/sh
 
-GTEST_BRANCHTAG=release-1.12.1
+GTEST_BRANCHTAG=release-1.12.1 # next is v1.14.0
```

Comment:
@cwschilly changing a dependency version should be requested in another issue. There were a discussion with @lifflander about that some weeks ago because I had envisaged to replace fetch script by a fetch action from CMakeLists.

Reply:
Sounds good!